日次CIエラー: TestE2E_OldCoreCommand

以下のエラーが発生している。

```console
------- Stdout: -------
=== RUN   TestE2E_OldCoreCommand
2022/04/01 04:04:49 step0: setup
2022/04/01 04:04:50 reloaded: "fake-store.json"
2022/04/01 04:04:50 reloaded: "fake-store.json"
2022/04/01 04:04:50 reloaded: "fake-store.json"
    e2e_test.go:73: context deadline exceeded
--- FAIL: TestE2E_OldCoreCommand (10.01s)
```

--- a/e2e/old_core_command/autoscaler.yaml
+++ b/e2e/old_core_command/autoscaler.yaml
@@ -1,0 +1,7 @@
+resources:
+  - type: ELB
+    name: "elb"
+    selector:
+      names: ["autoscaler-e2e-old-core-command"]
+autoscaler:
+  cooldown: 5

